dev/playbooks/roles/git/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+- name: ensure git is installed
+  yum: name=git state=present
+  become: yes
+  become_method: sudo
+
+- name: be sure git is configured
+  template: src=gitconfig.j2 dest={{ ansible_env.HOME }}/.gitconfig owner={{ ansible_user_id }}
```

Comment:
Got it, I am wondering if is is possible to have a global become and add false to this one since most of the tasks are ran by root. Anyway if you think that is not possible that is ok. Thanks for clarifying.